lib/paper_trail/model_config.rb

```diff
@@ -144,22 +144,35 @@ def setup_associations(options)
       # In PR _____ () we ask them to use `paper_trail_options` instead.
       @model_class.class_attribute :version_class_name
       @model_class.version_class_name = options[:class_name] || "PaperTrail::Version"
+      assert_concrete_activerecord_class(@model_class.version_class_name)
+
+      setup_has_many_versions(options)
+    end
+
+    def setup_has_many_versions(options)
+      unless options[:versions].is_a?(Hash)
+        options[:versions] = {
+          name: options[:versions]
+        }
+      end
```

Comment:
Should we add a deprecation warning here?

Reply:
Yes, just added. :)